camHeight: 720
camWidth: 1280
distortion: "plumb_bob"
distortionMat: [0.0960580262114993, -0.15729066603504493, -0.0007042382356700608, 0.0009266762323945099, 0.0]

rawIntrins:  [1075.7121338488246, 0.0, 575.1889512394235, 0.0, 1077.562204336173, 371.92157927267294, 0.0, 0.0, 1.0]
rectification: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection: [1090.7777099609375, 0.0, 575.5383247068385, 0.0, 0.0, 1093.9454345703125, 371.52561169551336, 0.0, 0.0, 0.0, 1.0, 0.0]